mw-webapp/src/logic/allWaysPage/allWaysTable/AllWaysTable.tsx

```diff
@@ -0,0 +1,42 @@
+import {useEffect, useState} from "react";
+import {getCoreRowModel, useReactTable} from "@tanstack/react-table";
+import {Table} from "src/component/table/Table";
+import {WayPreviewDAL} from "src/dataAccessLogic/WayPreviewDAL";
+import {columns} from "src/logic/allWaysPage/allWaysTable/columns";
+import {WayPreview} from "src/model/businessModelPreview/WayPreview";
+
+/**
+ * Render table of all ways preview
+ */
+export const AllWaysTable = () => {
```

Comment:
Maybe make it reusable and instead of getting ways inside, pass them as param? Then we can use it in OwnWays, Favourite ways and etc. If not we will have duplicate code

Reply:
TLDR: Make it WaysTable with WayPreview[] as param

Reply:
1. Sorry but we can't reusable any table with data because for each table will be different type of columns. We have reusable component Table. Doesn't it enough?
Maybe I didn't understood something?

2. About renaming. We have page AllWays so table should also be named AllWaysTable. Am I wrong?

Reply:
Colums can be same for AllWays, Owned Ways and etc, why not?

Reply:
What is the difference if we need to display WayPreview in all cases

Reply:
Yes we use one model nut the column can be difference/ I don't know. I didn't thought about tables in common yet. Created discuss in general)